app/models/ext_management_system.rb

```diff
@@ -158,14 +158,35 @@ def self.create_from_params(params, endpoints, authentications)
 
   serialize :options
 
-  supports :refresh_ems
+  supports_not :admin_ui
   supports_not :assume_role
-
-  supports :create_security_group do
-    unless SecurityGroup.class_by_ems(self).supports_create?
-      unsupported_reason_add(:create_security_group, _('Security Group creation is not supported'))
+  supports_not :block_storage
+  supports_not :change_password
+  supports_not :cloud_object_store_container_create
+  supports_not :cloud_subnet_create
+  supports_not :cloud_tenant_mapping
+  supports_not :console
+  supports_not :create_host_aggregate
+  supports     :create_security_group do
+    unless SecurityGroup.class_by_ems(self).supports?(:create)
+      unsupported_reason_add(:create_security_group, N_('Security Group creation is not supported'))
```

Comment:
I think this one you can just use `_()` because this block is evaluated at execution time.

Reply:
so, the UI would have to know which unsupported_reason string needs to be translated?  That puts a huge onus on the UI and is not very scalable, IMO.

Reply:
Either the UI should attempt to translate all strings, _or_, the methods themselves should do the translation, and the UI should assume translated.  We've done the latter with other things, though I"m not sure what the right and consistent answer is.

Reply:
@mzazrivec can you opine on what you believe the right approach is.

Reply:
these should all be `_()`